sdk/maps/Azure.Maps.Geolocation/src/Models/CountryRegion.cs

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using Azure.Core;
+
+namespace Azure.Maps.Geolocation
+{
+    /// <summary> The object containing the country/region information. </summary>
+    [CodeGenModel("CountryRegion")]
+    internal partial class CountryRegion
+    {
+        /// <summary> Initializes a new instance of CountryRegion. </summary>
+        internal CountryRegion()
+        {
+        }
+
+        /// <summary> Initializes a new instance of CountryRegion. </summary>
+        /// <param name="isoCode"> The IP Address&apos;s 2-character code [(ISO 3166-1)](https://www.iso.org/iso-3166-country-codes.html) of the country or region. Please note, IP address in ranges reserved for special purpose will return Null for country/region. </param>
+        internal CountryRegion(string isoCode)
+        {
+            IsoCode = isoCode;
+        }
+
+        /// <summary> The IP Address&apos;s 2-character code [(ISO 3166-1)](https://www.iso.org/iso-3166-country-codes.html) of the country or region. Please note, IP address in ranges reserved for special purpose will return Null for country/region. </summary>
```

Comment:
* Change `&apos;` to `'`
* Use C# style doctring for links: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/documentation-comments#d3-recommended-tags
Should be `<see href="url here"> ISO 3166-1 </see>`